src/lib/crypto.ts

```diff
@@ -0,0 +1,46 @@
+import crypto from "crypto";
+
+let algorithm = "aes-256-ctr";
+
+async function generateKey(secret: string): Promise<Buffer> {
+  return crypto.scryptSync(secret, "salt", 32);
+}
+
+async function encrypt({
+  encryptionKey,
+  text,
+}: {
+  encryptionKey: Buffer;
```

Comment:
In this file this is called `encryptionKey`, in `crypto.browser` is just `key`, they should have the same name, when they are being used in the KCDStrategy they are called as `encryptionKey` so the browser version would break. I like more just `key`, simpler.